universal-login-wallet/src/core/services/WalletStorageService.ts

```diff
@@ -0,0 +1,40 @@
+import {StorageEntry, StorageService} from './StorageService';
+import {WalletService} from '@universal-login/sdk';
+import {ApplicationWallet, ensure} from '@universal-login/commons';
+import {asObject, asString, Sanitizer} from '@restless/sanitizers';
+
+export class WalletStorageService {
+  private storage: StorageEntry<ApplicationWallet>;
+
+  constructor(
+    private walletService: WalletService,
+    storageService: StorageService,
```

Comment:
We should make a maximum length of a line of 120 characters. Simple function headers take to many lines.

Reply:
In my opinion, even with 2 parameters, this variant is easier to read, since every parameter occupies a single line, it's easy to distinguish names from types. And you can just skim through it to know what that constructor takes.